open-metadata-implementation/common-services/generic-handlers/src/main/java/org/odpi/openmetadata/commonservices/generichandlers/PortHandler.java

```diff
@@ -156,22 +156,22 @@ public String createPort(String              userId,
              */
             final String portGUIDParameterName = "portGUID";
 
-            this.linkElementToElement(userId,
-                                      null,
-                                      null,
-                                      processGUID,
-                                      processGUIDParameterName,
-                                      OpenMetadataAPIMapper.PROCESS_TYPE_NAME,
-                                      portGUID,
-                                      portGUIDParameterName,
-                                      OpenMetadataAPIMapper.PORT_TYPE_NAME,
-                                      false,
-                                      false,
-                                      supportedZones,
-                                      OpenMetadataAPIMapper.PROCESS_PORT_TYPE_GUID,
-                                      OpenMetadataAPIMapper.PROCESS_PORT_TYPE_NAME,
-                                      null,
-                                      methodName);
+            this.uncheckedLinkElementToElement(userId,
+                                               null,
+                                               null,
+                                               processGUID,
+                                               processGUIDParameterName,
+                                               OpenMetadataAPIMapper.PROCESS_TYPE_NAME,
+                                               portGUID,
+                                               portGUIDParameterName,
+                                               OpenMetadataAPIMapper.PORT_TYPE_NAME,
+                                               false,
```

Comment:
forLineage and forDuplicateProcessing should be parameters not passed as fixed values